src/main/java/com/htisolutions/poolref/entities/User.java

```diff
@@ -21,19 +23,25 @@
     private String nickname;
 
     @NotNull
-    private String hashedpassword;
+    private String hashed_password;
+
+    @Nullable
```

Comment:
If nullable is causing errors remove it, you are importing it from a strange library anyway